app/controllers/pages_controller.rb

```diff
@@ -1,5 +1,6 @@
 class PagesController < ApplicationController
   before_action :skip_session_expiration
+  skip_before_action :disable_caching
```

Comment:
Sure, although creating a shared example group shouldn't be too hard and does feel thematically like it could be a part of this PR, so I think it might make sense to do it now, too. We have a few shared example groups in this repo already.